util/linux_install.sh

```diff
@@ -18,9 +17,24 @@ elif grep ID /etc/os-release | grep -q arch; then
 	cd /tmp/dfu-programmer
 	makepkg -sic
 	rm -rf /tmp/dfu-programmer/
-
+elif grep ID /etc/os-release | grep -q gentoo; then
+  echo "This script will make a USE change in order to ensure that that QMK works on your system. All changes will be sent to the the file /etc/portage/package.use/qmk_firmware -- please review it, and read Portage's output carefully before installing any packages on your system. You will also need to ensure that your kernel is compiled with support for the keyboard chip that you are using (e.g. enable Arduino for the Pro Micro). Further information can be found on the Gentoo wiki."
+  echo -n "Proceed (y/N)? "
+  old_stty_cfg=$(stty -g)
+  stty raw -echo
+  answer=$( while ! head -c 1 | grep -i '[ny]' ;do true ;done )
+  stty $old_stty_cfg
+    if echo "$answer" | grep -iq "^y" ;then
+      sudo touch /etc/portage/package.use/qmkfirmware
+      echo "sys-devel/gcc multilib" | sudo tee --append /etc/portage/package.use/qmkfirmware > /dev/null
+      sudo emerge -auN sys-devel/gcc app-arch/unzip app-arch/zip net-misc/wget app-mobilephone/dfu-util sys-devel/crossdev dev-embedded/avrdude
```

Comment:
If I am not mistaken dev-embedded/dfu-programmer is needed to flash the firmware on many boards.